components/atom/videoPlayer/test/index.test.js

```diff
@@ -24,7 +24,7 @@ describe('AtomVideoPlayer', () => {
 
   it('should render without crashing', () => {
     // Given
-    const props = {}
+    const props = {src: ''}
```

Comment:
if there is no value (`undefined`) given to src it crushes? In that case plz assign a defaultValue to it